assets/css/style.css

```diff
@@ -459,6 +458,87 @@ header {
   color: #344563;
 }
 
+#project-popup {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background: #c1c7d0;
+  display: none;
+  padding: 50px;
+}
+
+.popup-wrapper {
+  background: #fff;
+  overflow: hidden;
+  height: 95%;
+  overflow-y: auto;
+  border-radius: 16px;
+  margin: auto;
+  padding: 20px;
+}
```

Comment:
Great design team 🤝

- [ ] As stated in the project details, kindly remove the horizontal scroll from both your mobile and desktop popups. This will make your design a replica of the Figma design.
- Your design
![mobile popup](https://github.com/Aqurds/mv-pmo/assets/63917725/3b985252-e2c5-4ca3-a365-84fcbe113814)
- Given design
![Given-mobile-popup](https://github.com/Aqurds/mv-pmo/assets/63917725/19b4f5d6-22da-4738-9aae-bd376209b825)

- [ ] I would recommend you increase the font weight of your text on both the popup windows so that your design can replicate the Figma design.
- [ ] I would also recommend you fix the image of the desktop popup. It appears blurred and this makes it appear different from the design provided in the Figma design.
- Your design
![desktop popup](https://github.com/Aqurds/mv-pmo/assets/63917725/bec1d4f3-e0ca-44b9-8a65-2a0391555162)
- Given design
![Given-desktop-popup](https://github.com/Aqurds/mv-pmo/assets/63917725/34af0ee7-9703-4426-be83-b739e74b0e03)

Reply:
@1GORDON, regards the scroll on a vertical angle, how the user will see the full information if it doesn't fit on the screen? I think you haven't provided me with perfect feedback on it. I will report it to Microverse.